rfcs/protocols/identity/network-identity-validation.md

```diff
@@ -0,0 +1,100 @@
+
+## Network Identity Validation
+
+* Authors: Venkatraman Ramakrishna, Krishnasuri Narayanam, Bishakh Chandra Ghosh
+* Status: Draft
```

Comment:
Can you change the status of all of the docs to "Proposed"? Once the PR is accepted, it means we have settled on the spec and are writing code to conform to that spec.

Reply:
Also, update the dates on all the docs (say, to today).